.github/workflows/ci.yml

```diff
@@ -334,6 +334,7 @@ jobs:
         make install-ingest-notion
         make install-ingest-delta-table
         pip install "unstructured[openai]"
+        pip install "unstructured[langchain]"
```

Comment:
I think ingest test imports are already covered by the `pip install "unstructured[openai]"` above this.

it looks like your test is added as a unit test, so the install here won't help. ultimately if this is now required for the unit tests to pass, probably means this needs to be required [via tests](https://github.com/Unstructured-IO/unstructured/blob/main/requirements/test.in) so that way anyone running the tests has it, right?

Reply:
Ok, so remove the pip install from ci.yml and add `langchain` to the test.in file?